go-controller/pkg/ovn/pods.go

```diff
@@ -238,31 +237,40 @@ func (oc *Controller) addLogicalPort(pod *kapi.Pod) error {
 
 	var podMac net.HardwareAddr
 	var podIfAddrs []*net.IPNet
-	var args []string
-	var addresses string
+	var cmds []*goovn.OvnCommand
+	var addresses []string
+
+	// Check if the pod's logical switch port already exists. If it
+	// does don't re-add the port to OVN as this will change its
+	// UUID and and the port cache, address sets, and port groups
+	// will still have the old UUID.
+	lsp, err := oc.ovnNBClient.LSPGet(portName)
+	if err != nil && err != goovn.ErrorNotFound && err != goovn.ErrorSchema {
+		return fmt.Errorf("unable to get the lsp: %s from the nbdb: %s", portName, err)
+	}
+
+	if lsp == nil {
+		lspAddCmd, err := oc.ovnNBClient.LSPAdd(logicalSwitch, portName)
+		if err != nil {
+			return fmt.Errorf("unable to create the LSPAdd command for port: %s from the nbdb", portName)
+		}
+		klog.V(5).Infof("Created LSPAdd cmd for port: %s", portName)
```

Comment:
Any reason to have a Infof() after this command creation, but not after all the others below?

Reply:
Perhaps left-over from a debugging session. Will get rid of it.